azure_arc_k8s_jumpstart/docs/local_kind.md

```diff
@@ -0,0 +1,160 @@
+  * If you are a MacOS User, install [Docker Desktop for Mac](https://docs.docker.com/docker-for-mac/).
+  * If you are a Linux user, use your package manager to install the [Docker engine](https://docs.docker.com/engine/install/).
+
+* Install the [Go programming language](https://golang.org/dl/).
+
+* Create Azure Service Principal (SP)   
+
+    To connect a Kubernetes cluster to Azure Arc, Azure Service Principal assigned with the "Contributor" role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+
+    ```bash
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```az ad sp create-for-rbac -n "http://AzureArcK8s" --role contributor```
+
+    Output should look like this:
+
+    ```
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcK8s",
+    "name": "http://AzureArcK8s",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+    
+    **Note**: It is optional but highly recommended to scope the SP to a specific [Azure subscription and Resource Group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) 
+
+* Enable subscription for two providers for Azure Arc enabled Kubernetes<br> 
+  Registration is an asynchronous process, and registration may take approximately 10 minutes.
+  ```bash
+  az provider register --namespace Microsoft.Kubernetes
+  Registering is still on-going. You can monitor using 'az provider show -n Microsoft.Kubernetes'
+
+  az provider register --namespace Microsoft.KubernetesConfiguration
+  Registering is still on-going. You can monitor using 'az provider show -n Microsoft.KubernetesConfiguration'
+  ```
+  You can monitor the registration process with the following commands:
+  ```bash
+  az provider show -n Microsoft.Kubernetes -o table
+ 
+  az provider show -n Microsoft.KubernetesConfiguration -o table
+  ```
+
+* Install the Azure Arc for Kubernetes CLI extensions ***connectedk8s*** and ***k8sconfiguration***:
+
+  ```bash
+  az extension add --name connectedk8s
+  az extension add --name k8sconfiguration
+  ```
+
+
+ 
+# Deployment
+
+* Install kind
+  * On Linux: 
+
+    ```bash
+    curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.9.0/kind-linux-amd64
+    chmod +x ./kind
+    sudo mv ./kind /usr/local/bin/kind
+    ```
+  * On Mac:
+
+    ```bash
+    brew install kind
+    ```
+  * On Windows:
+       ```powershell
+       choco install kind
+       ```
+
+
+* Navigate to the folder that has the kind cluster definition.
+  ```bash
+  cd azure_arc/azure_arc_k8s_jumpstart/kind
+  ```
+
+* Create the kind cluster
+    ```bash
+    kind create cluster --config kind_cluster.yaml
+    ```
+    ![](../img/kind_connect/01.png)
+
```

Comment:
add --name and --kubeconfig flags. 
Update screenshot accordingly

Reply:
will add

Reply:
Would you be OK with NOT adding --kube-config? The default should work for most people, and given we're making this universal across Windows/Linux/MacOS users, specifying the --kube-config flag would require another double command.

Reply:
Yes good point. Let's till have this as an explanation in a bullet point

Reply:
OK. Will update this.